### PR TITLE
[CS-3459] Add color picker modal in create profile flow

### PR DESCRIPTION
**Pull request by topwebtek7**
### Description

Added color picker modal behavior in create profile flow and did some clean up.

- [x] Completes #CS-3459

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

https://user-images.githubusercontent.com/16714648/160138897-76e887e0-db2c-4986-af7d-7bcc40fccd65.mp4

### CODE REVIEW

File: cardstack/src/screens/ColorPickerModal.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    params: { defaultColor = '#000000', onSelectColor },
+  } = useRoute<ColorPickerModalRouteType>();
+
+  const [selectedColor, setSelectedColor] = useState(defaultColor);
+
+  useEffect(() => {
+    Keyboard.dismiss();
+  }, []);
+
+  const onSelect = () => {
```
Comment (oidoug) [inline cardstack/src/screens/ColorPickerModal.tsx]: nit: wrap on useCallback

**Review by oidoug — APPROVED** [review]
The picker UI is pretty cool!
Just a small nit.

File: cardstack/src/screens/ColorPickerModal.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        borderTopLeftRadius={20}
+        borderTopRightRadius={20}
+        height="50%"
+        overflow="scroll"
+        backgroundColor="white"
+        paddingHorizontal={5}
+      >
+        <Container width="100%" alignItems="center" padding={5}>
+          <SheetHandle />
+        </Container>
```
Comment (danibonilha) [inline cardstack/src/screens/ColorPickerModal.tsx]: Doesn't using `<Sheet scrollEnabled={false} />` as the container works for this case?

**Review by danibonilha — CHANGES_REQUESTED** [review]
It's broken on android and on smaller devices

image

**Comment by topwebtek7** [comment]
Fixed ui in android, iphone SE -
iPhoneSE
Screen Shot 2022-03-28 at 15 24 02
Android
Screen Shot 2022-03-28 at 15 23 25

File: cardstack/src/screens/ColorPickerModal.tsx (hunk 2 above)
Comment (topwebtek7) [reply cardstack/src/screens/ColorPickerModal.tsx]: cool, updated with Sheet component

**Review by danibonilha — APPROVED** [review]
LGTM

**Pull request merged**